ethcore/src/evm/evm.rs

```diff
@@ -95,11 +95,17 @@ impl<'a> Finalize for Result<GasLeft<'a>> {
 	}
 }
 
+/// Cost calculation type. For low-gas usage we calculate costs using usize instead of U256
```

Comment:
Why? If it doesn't fit in `usize` we fallback to U256 - it optimizes for u32 on 32-bit systems and for u64 for 64-bit ones.
Using optimized (whether it's u64 or u32) and non-optimized (u256) version should is not affecting consensus.